packages/app/src/systems/Tweet/components/TwitterDialog.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect } from "react";
+
+import { useTwitterDialog } from "../hooks/useTwitterDialog";
+
+import { Button, Dialog, Link, useDialog } from "~/systems/UI";
+
+// const tweetText = `I'm using #SwaySwap, a blazingly fast DEX built on Fuel @fuellabs_\nhttps://fuellabs.github.io/swayswap`;
```

Comment:
this will remain with this fake version?

Reply:
We are going to hold until we have a final text

Reply:
The text is final. To clarify, my comment about weren't about the finality of the text, but rather to not have a running version of SwaySwap with the above text that someone might Tweet by accident before we're ready.